zipkin-storage/cassandra/src/main/java/zipkin2/storage/cassandra/internal/SessionBuilder.java

```diff
@@ -79,7 +79,7 @@ public static CqlSession buildSession(
     if (useSsl) config = config.withClass(SSL_ENGINE_FACTORY_CLASS, DefaultSslEngineFactory.class);
 
     // Log categories can enable query logging
-    Logger requestLogger = LoggerFactory.getLogger(RequestLogger.class);
+    Logger requestLogger = LoggerFactory.getLogger(SessionBuilder.class);
```

Comment:
Calling out this change and the `Schema -> DefaultSessionFactory` above as needing a quick look in case this _was_ intentional.